CODEOWNERS

```diff
@@ -364,7 +364,7 @@ tests/knative-build @rakesh-garimella @lilitgh @k15r @nachtmaar @montaro @marcob
 /tests/end-to-end/kubeless-integration/ @Abd4llA @joek @abbi-gaurav @rakesh-garimella @lilitgh @k15r @nachtmaar @anishj0shi @montaro @marcobebway @radufa @sayanh @antoineco
 
 # external-solution-integration test directory
-/tests/end-to-end/external-solution-integration @sjanota @janmedrek @rakesh-garimella @sayanh
+/tests/end-to-end/external-solution-integration @sjanota @janmedrek @rakesh-garimella @sayanh @aszecowka @crabtree @pkosiec @kfurgol @tgorgol @dbadura @kjaksik @akgalwas @Szymongib @franpog859 @Maladie
```

Comment:
@sjanota Should we address the issue of this directory having only 4 codeowners? Perhaps the responsibility for maintaining this should be split across all the teams that have their tests here?

Reply:
Whoa. Why whole teams? I'm sure not everyone in the team has touched this piece of code. I think we should shrink it a little.

Reply:
Agreed. I'll be adding me, @dbadura, @janmedrek and @Szymongib to codeowners.